docs/javascript/v2/get-started/react-sample-app-quickstart.mdx

```diff
@@ -0,0 +1,179 @@
+
+    ![token endpoint](https://user-images.githubusercontent.com/11087313/140727818-43cd8be4-b3bf-4b34-9921-a77f9a1b819d.png)
+
+    **Example**:
+
+    ```bash
+    REACT_APP_TOKEN_GENERATION_ENDPOINT = 'https://prod-in2.100ms.live/hmsapi/example.app.100ms.live/'
+    ```
+
+## Test the app
+
+-   Start the app with the below command. The app should now be running at [http://localhost:3000/](http://localhost:3000/). You should see a welcome message saying, "Almost There!".
+
+    ```bash section=runApp sectionIndex=1
+    yarn start
+    ```
+
+-   To test audio/video functionality, you need to connect to a 100ms room; please check following steps for the same:
+
+    1. Navigate to your [100ms Dashboard](https://dashboard.100ms.live/dashboard) or [create an account](https://dashboard.100ms.live/register) if you don't have one.
+    2. To test this app quickly, use the "Video Conferencing Starter Kit" to create a room with a default template assigned to it.
+    3. Go to [Rooms page](https://dashboard.100ms.live/rooms) in your Dashboard, copy the "Room Id" of the room and role created in the above step.
+    4. Add the "Room Id" and role to the localhost URL to test the app. For example, `http://localhost:3000/633fcdd84208780bf665346a/host`
+
+    <br />
+    <video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+        <source src="/docs/guides/join-room-sample-app.mp4" type="video/mp4" />
+    </video>
+    <br />
+
+## Customize your app
+
+This sample app allows UI customization, like updating the UI layout, logo, tile aspect ratio, theme, color, etc. To make these customizations, you can use various environment variables in the env file.
+
+### Configure UI layout
+
+-   You can customize the layout to arrange the position of video tiles based on roles. The value of `REACT_APP_POLICY_CONFIG` should be a JSON stringified object of the structure:
+
+    ```ts section=configureUi sectionIndex=1
+    type HMSRoleName = string;
+
+    /**
+     * undefined means none
+     * empty array [] for all roles
+     */
+    interface RoleConfig {
+        center?: HMSRoleName[];
+        sidepane?: HMSRoleName[];
+        selfRoleChangeTo?: HMSRoleName[];
+        remoteRoleChangeFor?: HMSRoleName[];
+    }
+
+    interface PolicyConfig {
+        [role: string]: RoleConfig;
+    }
+    ```
+
+-   **Example**: If you wish to keep the video tile of the student in the center for the trainer and his video tile on the side pane, you can use the below JSON stringified object.
+
+    ```bash
+    REACT_APP_POLICY_CONFIG = "{"trainer":{"center": ["student"],"sidepane":["trainer"]}}"
+    ```
+
+### Change logo
+
+You can upload your logo in the sample app to customize the UI for your brand. You can achieve this in two ways:
+
+-   Using the no-code editor in the UI
```

Comment:
this is only used if you have a deployed sample app and cannot be used if you have the cloned repo.

Reply:
got it, removed this info.